ios/Makefile

```diff
@@ -0,0 +1,20 @@
+SWIFT_RUN=swift run -c release --package-path Tools
+WORKSPACE=DroidKaigi2021.xcworkspace
+
+.PHONY: open
+open:
+	@open $(WORKSPACE)
+
+# SwiftLint
+.PHONY: run-swiftlint
+run-swiftlint:
+	$(SWIFT_RUN) swiftlint
+
+# SwiftGen
+.PHONY: run-swiftgen
+run-swiftgen:
+	./scripts/swiftgen.sh
+
+.PHONY: run-swiftgen-generate-xcfilelists
```

Comment:
I don't think this command is needed. This command is used only when setup configuration which is done.

Reply:
I left this command just in case. For example, in case  adding new Localizable.strings file.
I'm thinking of doing tasks for localization ​​with other PR. 🙏🏼